splinterd/src/daemon.rs

```diff
@@ -194,7 +194,7 @@ impl SplinterDaemon {
                 }
 
                 return Err(StartError::StorageError(format!(
-                    "Run the `splinter upgrade` command to update outdated state files to
+                    "Run the `splinter upgrade` command to update outdated state files to\
```

Comment:
this needs a space after to: to\ => to \

Otherwise you get this output `files toa Splinter`

Reply:
Fixed!